src/main/webapp/app/exercises/text/assess/textblock-feedback-editor/textblock-feedback-editor.component.html

```diff
@@ -60,50 +60,113 @@
+                    <ul *ngIf="criteria.length == 1" class="dropdown-submenu-single" id="test" ngbDropdownMenu>
+                        <li *ngFor="let instruction of criteria[0]!.structuredGradingInstructions">
+                            <a (click)="updateAssessmentWithDropdown(instruction)" class="dropdown-item" [style.background-color]="setInstrColour(instruction)">
+                                <table class="table layout-table">
+                                    <tbody>
+                                        <tr>
+                                            <td class="td-score fw-bold">{{ instruction!.credits }}P</td>
+                                            <td>
+                                                <span class="td-instruction">{{ instruction!.instructionDescription }}</span>
+                                            </td>
+                                            <td class="td-usage-count">
+                                                <span *ngIf="instruction!.usageCount && instruction!.usageCount !== 0; else usageCountInfinity">{{ instruction!.usageCount }}</span>
+                                                <ng-template #usageCountInfinity>&#8734;</ng-template>
+                                                <jhi-help-icon placement="top" text="artemisApp.exercise.usageCountHint"></jhi-help-icon>
+                                            </td>
+                                        </tr>
+                                    </tbody>
+                                </table>
+                            </a>
+                        </li>
+                    </ul>
+                    <!--if there is more than one criterion, the dropdown-menu contains the criteria titles and submenu the corresponding instructions-->
+                    <ul *ngIf="criteria.length > 1" class="dropdown-menu" ngbDropdownMenu>
+                        <li *ngFor="let criterion of criteria">
+                            <a class="dropdown-item" *ngIf="criterion.title; else withoutTitle"
+                                >{{ criterion.title }}&nbsp;<fa-icon class="float-end" [icon]="faAngleRight"></fa-icon
+                            ></a>
+                            <ng-template #withoutTitle
+                                ><a class="dropdown-item">(No title) <fa-icon class="float-end" [icon]="faAngleRight"></fa-icon></a
+                            ></ng-template>
+                            <ul class="dropdown-menu dropdown-submenu flex-column flex-wrap">
+                                <li *ngFor="let instruction of criterion!.structuredGradingInstructions">
+                                    <a (click)="updateAssessmentWithDropdown(instruction)" class="dropdown-item" id="item" [style.background-color]="setInstrColour(instruction)">
+                                        <table class="table layout-table">
```

Comment:
This table-section is duplicated from above, can this be avoided somehow?

Reply:
I know :/ but not that I know of.

Reply:
@Strohgelaender @Dominik-Weinzierl I updated this pr and provide the duplicated table-section within an additional component now.